src/app/components/NoPreview/index.tsx

```diff
@@ -12,21 +10,15 @@ export const NoPreview: FC<NoPreviewProps> = ({ placeholderSize }) => {
   const { t } = useTranslation()
 
   return (
-    <Box
-      gap={4}
-      sx={{
+    <div
+      className="flex flex-1 flex-col items-center justify-center gap-4 text-gray-500 font-semibold bg-gray-100"
```

Comment:
- let's use text-muted-foreground instead of tailwind text-gray-500 
- why  bg-gray-100 was added ?